vector/src/main/java/im/vector/app/features/spaces/SpaceListViewModel.kt

```diff
@@ -278,31 +278,23 @@ class SpaceListViewModel @AssistedInject constructor(@Assisted initialState: Spa
             displayName = QueryStringValue.IsNotEmpty
         }
 
-        val flowSession = session.flow()
-
         combine(
-                flowSession
-                        .liveUser(session.myUserId)
-                        .map {
-                            it.getOrNull()
-                        },
-                flowSession
+                session.flow()
                         .liveSpaceSummaries(params),
-                session
-                        .accountDataService()
+                session.accountDataService()
                         .getLiveRoomAccountDataEvents(setOf(RoomAccountDataTypes.EVENT_TYPE_SPACE_ORDER))
                         .asFlow()
-        ) { _, communityGroups, _ ->
```

Comment:
to double check, were the other flows being used as some sort of predicate? / Do we need `.getLiveRoomAccountDataEvents(...).asFlow()`

it's a bit strange to see us submit flows to `combine` and then ignore their result :thinking:

Reply:
It's true that we ignore the result, but we want to be notified when the account data `EVENT_TYPE_SPACE_ORDER` is updated.